src/src/radio.c

```diff
@@ -1220,8 +1216,6 @@ int8_t otPlatRadioGetReceiveSensitivity(otInstance *aInstance)
 #if OPENTHREAD_CONFIG_MAC_HEADER_IE_SUPPORT
 void nrf_802154_tx_started(const uint8_t *aFrame)
 {
```

Comment:
We may consider moving the whole implementation of `nrf_802154_tx_started()` to `SubMac`, and `SubMac` provide a function for radio driver to generate the ACK frame. This could greatly reduce platform radio APIs and the complexity of the `otRadioFrame`.

Reply:
I was considering wrapping all manipulations on SFD in a single callback. We probably shouldn't put it into SubMac, as this callback is probably called by a ISR.